routes/api.py

```
@@ -0,0 +1,512 @@
+                            related_resources['Ingresses'].append({
+                                'name': ingress.metadata.name,
+                                'status': 'Active',
+                                'info': 'Using Secret for TLS'
+                            })
+                            
+        elif resource_type == 'service':
+            resource = v1.read_namespaced_service(name, namespace)
+            resource_details = {
+                'type': 'Service',
+                'name': resource.metadata.name,
+                'type': resource.spec.type,
```

Ruff (0.8.0)

Dictionary key literal `'type'` repeated

(F601)